packages/sit-onyx/src/components/OnyxFilterTag/OnyxFilterTag.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts" setup>
+import xSmall from "@sit-onyx/icons/x-small.svg?raw";
+import { SKELETON_INJECTED_SYMBOL, useSkeletonContext } from "../../composables/useSkeletonState";
+import OnyxTag from "../OnyxTag/OnyxTag.vue";
+import type { OnyxTagProps } from "../OnyxTag/types";
+
+const props = withDefaults(
+  defineProps<
+    Omit<OnyxTagProps, "color" | "icon" | "clickable"> & {
+      /**
+       * The label displayed on the Tooltip
+       */
+      clickable: string;
+    }
+  >(),
```

Comment:
Would be nice to make the component more specific to the filter use-case:
```suggestion
  defineProps<
    Omit<OnyxTagProps, "color" | "icon" | "clickable"> & {
      /**
       * The text content of the tag.
       */
      label: string;
      /**
       * An icon which will be displayed on the left side of the tags label.
       */
      icon?: string;
      /**
       * If `true` the filter is selected, shows an 'x' icon and can be removed on click.
       */
      active: boolean;
    }
  >(),
  // ...
  const tooltipLabel = computed(() => props.active ? t.value('filter-tag.click-to-remove', { label }) : t.value('filter-tag.click-to-add', { label }));
```